ark-plonk-book/src/chapter_1.md

```diff
@@ -0,0 +1,373 @@
+
+Where $\phi(\alpha)$ is an evaluation of the polynomial $\phi$ at point $\alpha$, $\phi(x)=\sum_{j=0}^{t}\phi_jx^j$ and $t=deg(\phi)$. 
+ 
+3. **Open:** The prover outputs the polynomial $\phi(x)$ used to create the commitment.
+
+
+4. **VerifyPoly:** The verifier checks if $C$ is indeed a commitment to $\phi(x)$ as verifying if $C$ is equal to $[\phi(\alpha)]$. The verifier is able to do so since: $$[\phi(\alpha)]=\prod_{j=0}^{t}([\alpha^j])^{\phi_j}$$ and $⟨[1], [\alpha], [\alpha^2],.........,[\alpha^t]⟩$ is known to the verifier.
+
+
+5. **CreateWitness:** The prover outputs $<\beta,\phi(\beta),w_{\beta}>$, where $w_{\beta}$ is a witness for the evaluation $\phi(\beta)$ of $\phi(x)$ at index $\beta\neq \alpha$ chosen by the verifier. $$w_{\beta}=g^{\Psi_{\beta}(\alpha)}$$  and $$\Psi_{\beta}(x)=\dfrac{\phi(x)-\phi(\beta)}{x-\beta}(x)$$
+
+The construction of $\Psi$ is based on an algebraic property of polynomials which guarantees that $\psi(x)-\psi(\beta)$ is divisible by $x- \beta$ for any $\beta\in F_p$.
+ 
+6. **VerifyEval:** In this last step, the verifier checks if $\phi(\beta)$ is indeed the  evaluation  of $\phi(x)$ at index $\beta$ by checking if the following holds: $$e(C,g)=e(g^{\Psi_{\beta}(\alpha)},g^{\alpha}.g^{-\beta})e(g,g)^{\phi(\beta)}$$
+   
+
+## Zero Knowledge Proof systems
+
+Before we start talking about zero knowledge proof systems, let's first define what is a proof system:\
+A proof system is a protocol by which one party (prover) wants to convince another party (verifier) that a given statement is true. 
+
+### Zero knowledge proof system:
+In zero-knowledge proofs, the prover convinces the verifier about the truthfulness of the statement without revealing any information about the statement itself.
+ 
+#### Properties
+A zero-knowledge proof needs to fulfill each of the following properties to be fully described:
+* **Completeness:**  An honest prover is always able to convince the verifier of the truthfulness of their claim.
+* **Soundness:** If the prover’s claim is false (malicious prover), the verifier is not convinced.
+* **Zero knowledge:** the proof should not reveal any information to the verifier beyond the truthfulness of the given claim.
+
+#### Types of Zero knowledge proofs
+
+There are two types of zero knowledge proofs (interactive and non interactive ones)
+Interactive zero-knowledge proof systems were first introduced in 1985 by Goldwasser, Micali and Rackoff. Non-interactive schemes were introduced later on by Blum et al. The main difference between both schemes is that interactive proofs require interaction between both parties which means both have to be online in order to do so which could be seen as inconvenient especially for modern cryptography applications while non-interactive proofs need a trusted setup preprocessing phase instead.
```

Comment:
Non-interactive protocols don't necessarily require trusted setup, they just require shared setup, where everyone knows which statement is being proved and what protocol is being used.